en/collect/import-using-online-bibliographic-database.md

```diff
@@ -20,123 +22,177 @@ However, it is still possible to import hundreds or even thousands of entries fr
 
 If you need to use an HTTP proxy server, you can configure JabRef to use a proxy using the "Network" preferences \(**Options → Preferences → Network**\).
 
+## Search Syntax
+
+The search syntax is adapted from [Apache Lucene](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html).
```

Comment:
That we use Lucene shouldn't matter for the average user, thus I would remove all references to it.

Reply:
I think, for researchers in the field of computer science, it is of interest. I will rewrite that part.

Reply:
You mean people from computer science are the only ones that might actually understand the sentence ;-) Please remember this is a help page and not an reference for the inner workings of JabRef.